lib/archethic/networking/scheduler.ex

```diff
@@ -139,4 +130,20 @@ defmodule Archethic.Networking.Scheduler do
       {:ok, p2p_port, web_port}
     end
   end
+
+  defp handle_new_ip(%Transaction{address: tx_address}) do
+    nodes =
+      P2P.authorized_and_available_nodes()
+      |> Enum.filter(&Node.locally_available?/1)
+      |> P2P.nearest_nodes()
+
+    Utils.await_confirmation(tx_address, nodes)
+
+    types = [:node, :oracle, :node_shared_secrets, :reward]
+
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, types, fn type ->
```

Comment:
After https://github.com/archethic-foundation/archethic-node/pull/933 merge,  you'd need to use `NetworkChain.asynchronous_resync_many(network_chain_types)` instead.